scenes/cslib_number.txt

```diff
@@ -118,6 +118,52 @@
 *goto min_loop
 
 
+*comment MAX_STAT
+*comment ------------------------------
+*comment Returns the largest of a set of given stats.
+*comment ------------------------------
+*comment 	params:
+*comment		param_n(string): a stat (passed by name)
+*comment	returns: 
+*comment		(string): the stat with the largest value
+*label max_stat
+*params
+*gosub_scene cslib_assert greater_or_equal_to param_count 2
+*temp current param[1]
+*temp n 2
+*label max_stat_loop
+*if (n > param_count)
+	*set cslib_ret current
+	*return
+*if ({param[n]} > {current})
+	*set current param[n]
+*set n + 1
+*goto max_stat_loop
+
+
+*comment MIN_STAT
+*comment ------------------------------
+*comment Returns the smallest of a set of given stats.
```

Comment:
Returns the variable name of the smallest of a set of given stats.

Reply:
done